tpl/bootstrap.php

```diff
@@ -379,31 +379,46 @@
 if (strlen($NOTICE)):
 ?>
 				<div role="alert" class="alert alert-info">
-					<span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span><span class="pl-1"><?php echo htmlspecialchars($NOTICE), PHP_EOL; ?></span>
+					<span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+					<?php echo htmlspecialchars($NOTICE), PHP_EOL; ?>
 				</div>
 <?php
 endif;
 ?>
 				<div id="remainingtime" role="alert" class="hidden alert alert-info">
-					<span class="glyphicon glyphicon-fire" aria-hidden="true"></span><span class="pl-1"></span>
+					<span class="glyphicon glyphicon-fire" aria-hidden="true"></span>
```

Comment:
Re-add the spans please. (see other comment) It will break when the JSON is not loaded otherwise.